kafka/src/main/java/com/bazaarvoice/emodb/kafka/metrics/DropwizardMetricsReporter.java

```diff
@@ -3,22 +3,24 @@
 import com.codahale.metrics.Gauge;
 import com.codahale.metrics.MetricRegistry;
 import com.codahale.metrics.SharedMetricRegistries;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
 import org.apache.kafka.common.MetricName;
 import org.apache.kafka.common.metrics.KafkaMetric;
 import org.apache.kafka.common.metrics.MetricsReporter;
 import org.coursera.metrics.datadog.TaggedName;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
 public class DropwizardMetricsReporter implements MetricsReporter {
 
+    public static final String REGISTRY_NAME = "default";
```

Comment:
Is the diff in this file necessary?

Reply:
Oh, not anymore; I had do to this to make the constant available so I could send metrics  to the same registry from a static class, which I've since removed.